portfolio/src/main/webapp/slideShowScript.js

```diff
@@ -1,3 +1,2 @@
-
   showSlides();
-  setTimeout(showSlidesAuto, 10000); // Change image every 5 seconds
+  setInterval(showSlidesAuto, 10000); // Change image every 10 seconds
```

Comment:
(Optional): To avoid letting the comment go out of date in the future, consider re-writing the code to not need comments at all. Something like `setInterval(showSlidesAuto, 10 * 1000);` more clearly shows that the function will get called every 10 seconds.

Reply:
I hear your concern here, I think I'm gonna change this too for PR 7